wrappers/rust/oneds-telemetry/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "oneds-telemetry"
+version = "0.1.0"
+edition = "2021"
+publish = false
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+chrono = { version = "0.4.30" }
+log = { version = "0.4.20" }
+once_cell = "1.12.0"
```

Comment:
Is there a reason to use `once_cell`, now that `Once` and `OnceLock` are part of the Rust standard library?